src/databaseService/index.js

```diff
@@ -0,0 +1,59 @@
+import 'react-native'
+import * as firebase from 'firebase'
+
+const set = async (refPath, data) => {
+  try {
+    await firebase.database().ref(refPath).set(data)
+  } catch (error) {
+    console.log(error.message)
+  }
+}
+
+const get = async (refPath, mode = 'value') => {
+  let values = null
```

Comment:
I think you don't need this variable. You can just return `databaseSnapshot.val()`.